### ISSUE TITLE
wdio-wait-for functions' parameter types became incompatible with $ command return type

### ISSUE THREAD

**Issue by ferike86** [webdriverio/wdio-wait-for#120]
After this PR [#7978](https://github.com/webdriverio/webdriverio/pull/7978) in `webdriverio` project included in this release [v7.16.15 (2022-02-10)](https://github.com/webdriverio/webdriverio/blob/main/CHANGELOG.md#v71615-2022-02-10) all the `wdio-wait-for` functions' element type are incompatible with the `$` command return type. See an example:

`await browser.waitUntil(visibilityOf($('element')));`

```
Argument of type 'ChainablePromiseElement<Element>' is not assignable to parameter of type 'string | ChainablePromiseElement<Promise<Element>>'.
  Type 'ChainablePromiseElement<Element>' is not assignable to type 'ChainablePromiseElement<Promise<Element>>'.
    Type 'Element' is missing the following properties from type 'Promise<Element>': then, catch, finally, [Symbol.toStringTag]ts(2345)
```

With that change the return type of the `$` command has been changed from `ChainablePromiseElement<Promise<Element>>` to `ChainablePromiseElement<Element>`.

**Comment by christian-bromann** [comment]
Thanks for reporting!

Are you interested taking a stab fixing the issue? I am happy to help along the way.

**Comment by elaichenkov** [comment]
Fixed it in [2.2.5](https://github.com/webdriverio/wdio-wait-for/releases/tag/2.2.5). Could you please confirm that everything works as expected for you?
